Sources/UI/Style/AvatarViewStyle.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  AvatarViewStyle.swift
+//  StreamChat
+//
+//  Created by Alexey Bukhtin on 06/01/2020.
+//  Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+/// Avatars style.
+public struct AvatarViewStyle: Hashable {
+    /// An avatar radius.
+    public let radius: CGFloat
+    /// A placeholder font.
+    public let placeholderFont: UIFont?
+    
+    /// A double value of `radius`.
+    public var size: CGFloat {
```

Comment:
This is confusing: a circle's size is not double it's radius, so what is this value? Edge length?

Reply:
Avatars are rounded with some radius. Size is for width and height of it.

Reply:
Can we rename it so it's clearer? Something like `squareEdge` or `squareEdgeLength` or `boundingSquareEdge`?

Reply:
It's more than clear :) It's using a lot in other projects. It's actually diameter, but mostly id named as just size.